### DISCUSSION TITLE
How can I override primary color in Prime React?

### DISCUSSION THREAD

**Discussion by skj-skj** [primefaces/community#1226] [PrimeReact] [answered]
one way is to copy the css theme and update the primary color in it.
is there any better way which requires minimal changes?

edit:
--primary-color variable is not referenced in the css file.
I have replaced all occurrence of the hexcode mentioned in --primary-color with lets say red color.

**Comment by melloware** [comment]
Have you tried overriding the CSS root variable `--primary-color`.

![image](https://github.com/primefaces/community/assets/4399574/84b597f0-d8e2-498c-98c9-62c261607c37)

**Reply by skj-skj** [reply]
yes, but when I checked the css file that variable is not referenced. so overriding doesn't do anything.
this may be because css is compiled from sass and I don't have any idea how sass works.

**Reply by skj-skj** [reply, marked as answer]
found it, customisation of themes is mentioned here https://primereact.org/theming/ under Theme SCSS section.
